addon/components/course-publicationcheck.js

```diff
@@ -1,29 +1,19 @@
 /* eslint-disable ember/no-computed-properties-in-native-classes */
 import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
-import { tracked } from '@glimmer/tracking';
-import { computed } from '@ember/object';
-import { restartableTask } from 'ember-concurrency';
 
 export default class CoursePublicationCheckComponent extends Component {
   @service router;
-  @tracked objectivesRelationship;
 
-  @computed('objectivesRelationship.@each.programYearObjectives')
   get showUnlinkIcon() {
-    if (!this.objectivesRelationship) {
+    if (!this.args.course.courseObjectives) {
       return false;
     }
-    const objectivesWithoutParents = this.objectivesRelationship.filter((objective) => {
+    const objectivesWithoutParents = this.args.course.courseObjectives.filter((objective) => {
```

Comment:
This worked before so I left it this way, but I think I'll follow up with a PR to resolve this relationship before using it.

Reply:
:heavy_check_mark:

Reply:
Changed in https://github.com/ilios/common/pull/2471